gnomad_qc/v5/data_ingestion/federated_validity_checks.py

```diff
@@ -197,22 +259,19 @@ def main(args):
                 "Filtering to %d partitions and sex chromosomes.", test_n_partitions
             )
             test_ht = ht._filter_partitions(range(test_n_partitions))
-            sex_ht = hl.filter_intervals(
-                ht,
-                [hl.parse_locus_interval("chrX")] + [hl.parse_locus_interval("chrY")],
-            )
-            ht = test_ht.union(sex_ht)
 
-        # Read in parameters from config file.
-        with hl.hadoop_open(config_path, "r") as f:
-            config = json.load(f)
+            x_ht = hl.filter_intervals(
+                ht, [hl.parse_locus_interval("chrX")]
+            )._filter_partitions(range(test_n_partitions))
+
+            y_ht = hl.filter_intervals(
+                ht, [hl.parse_locus_interval("chrY")]
+            )._filter_partitions(range(test_n_partitions))
+
+            ht = test_ht.union(x_ht, y_ht)
 
         # Validate config file against schema.
-        try:
-            validate(instance=config, schema=schema)
-            logger.info("JSON is valid.")
-        except ValidationError as e:
-            raise ValueError(f"JSON validation error: %s, {e.message}")
+        config = validate_config(ht=ht, config_path=config_path, schema=schema)
 
         missingness_threshold = config["missingness_threshold"]
```

Comment:
Similarly, since these are only used once, can just reference the config dict in the function call below instead of declaring the variables here. I think it reads easier. This comment applies through L225